for/for/ViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ViewController.swift
+//  for
+//
+//  Created by Risako Kato on 2022/07/16.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet var transportationLabel: UILabel!
+    @IBOutlet var menuButton: UIButton! {
+        didSet {
+            var menuItems = [UIAction]()
+            for transportation in Transportation.allCases {
+                menuItems.append(UIAction(title: transportation.rawValue) { _ in
+                    self.transportationLabel.text = "\(transportation.rawValue)が選択されました"
```

Comment:
クロージャの中でselfをキャプチャする場合は循環参照を防ぐために`[weak self]`を使って弱参照にしてあげると良きです

```suggestion
                menuItems.append(UIAction(title: transportation.rawValue) { [weak self] _ in
                    self.transportationLabel.text = "\(transportation.rawValue)が選択されました"
```